commit c7ebd25e6a77ec997a62263b59f618957a0a409a
Author: Daniel Kolesa <daniel@octaforge.org>
Date:   Mon Jan 17 04:43:36 2022 +0100

    remove arabic from LINGUAS
    
    For some reason the desktop-docs/gpl docbook file fails to
    generate, which aborts the build, so disable for now.

diff --git a/desktop-docs/gpl/LINGUAS b/desktop-docs/gpl/LINGUAS
index dcebd12..f392a2b 100644
--- a/desktop-docs/gpl/LINGUAS
+++ b/desktop-docs/gpl/LINGUAS
@@ -1,4 +1,3 @@
-ar
 ca
 cs
 de
